src/core/agent/agent.types.ts

```diff
@@ -98,6 +98,9 @@ interface KeriaNotificationMarker {
 interface AgentServicesProps {
   signifyClient: SignifyClient;
   eventService: EventService;
+  identifierStorage: IdentifierStorage;
+  credentialStorage: CredentialStorage;
+  peerConnectionStorage: PeerConnectionStorage;
```

Comment:
I think something went wrong post merge here. The agent props only include `signifyClient` and `eventService` now. The other storage types are manually passed to services if they are required, otherwise there are too many being passed.

Reply:
@iFergal  The `identifierStorage` and `credentialStorage` come from the previous merges. Do you want me to clean them?

Reply:
@Sotatek-HocNguyena If you check develop, they are not part of agent services props - https://github.com/cardano-foundation/cf-identity-wallet/blob/develop/src/core/agent/agent.ts#L194